Dockerfile

```diff
@@ -0,0 +1,32 @@
+# ---- Base Alpine with installed Node ----
+FROM alpine:3.14.3 AS base
+
+# install node
+RUN apk add --update nodejs npm nghttp2
```

Comment:
Shouldn't we need to specify which version of node and npm we wanna use? We should be consistent with the rest of CI scripts.

Reply:
Ok, I will update it :)